### PR TITLE
chore(docs): Adding Playwright docs

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds the playwright docs

## Changes

- Adds playwright integration docs

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Screenshots

localhost_3000_tools-and-integrations_playwright

### CODE REVIEW

File: docs/docs/tools-and-integrations/playwright.mdx [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+    },
+  ],
+});
+```
+
+### The `home.spec.ts` Test Script
+
+The `home.spec.ts` test script contains three different tests based on the Pokeshop Demo UI fetures, which are:
+
+1. Creates a Pokemon
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/playwright.mdx:107]:
```suggestion
1. Create a Pokemon
```

File: docs/docs/tools-and-integrations/playwright.mdx [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+2. Imports a Pokemon (using an async process)
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/playwright.mdx:108]:
```suggestion
2. Import a Pokemon (using an async process)
```

File: docs/docs/tools-and-integrations/playwright.mdx [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+3. Deletes a Pokemon
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/playwright.mdx:109]:
```suggestion
3. Delete a Pokemon
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

File: docs/docs/tools-and-integrations/playwright.mdx [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+  - end to end testing
+  - end-to-end testing
+  - integration testing
+image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1698686403/docs/Blog_Thumbnail_14_rsvkmo.jpg
+---
+
+[Tracetest](https://tracetest.io/) is a testing tool based on [OpenTelemetry](https://opentelemetry.io/) that permits you to test your distributed application. It allows you to use the trace data generated by your OpenTelemetry tools to check and assert if your application has the desired behavior defined by your test definitions.
+
+[Playwright](https://playwright.dev/) is an open-source automation framework developed by Microsoft that enables cross-browser automation for web applications. It provides a set of APIs and libraries for automating interactions with web browsers such as Chrome, Firefox, and Microsoft Edge.
+
```
Comment (kdhamric) [inline docs/docs/tools-and-integrations/playwright.mdx:21]: We say what Tracetest and Playwright are, and then we say 'Why is this important?', but we have not spelled out what 'this' is yet. Should we add a paragraph that says:

The Tracetest integration for Playwright enables your current Playwright tests to easily capture a full distributed trace from your OpenTelemetry instrumented frontend and backend system. You can embed a Tracetest in this Playwright test, and allow trace-based testing assertions to be applied across this entire flow, enabling true end to end tests across your entire system.

File: docs/docs/tools-and-integrations/playwright.mdx [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+  @tracetest/playwright-->-Scripts: Ok
+  Scripts->>+Scripts: Executes test
+  Scripts->>+@tracetest/playwright: Runs test
+  @tracetest/playwright-->-Scripts: Ok
+  Scripts->>@tracetest/playwright: Waits for results and shows the summary
+```
+
+## The `@tracetest/playwright` npm Package
+
+The [`@tracetest/playwright`](https://www.npmjs.com/package/@tracetest/playwright) npm package is a Playwright plugin that allows you to run Trace-based testing using Tracetest and Playwright. It is a wrapper around the Tracetest API that allows you to configure, orchestrate and run tests from Playwright.
```
Comment (kdhamric) [inline docs/docs/tools-and-integrations/playwright.mdx:46]: I do not think 'Trace-based testing' should be capitalized

**Review by kdhamric — APPROVED** [review]
Left a couple comments - looks good

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**